code/modules/clothing/gloves/color.dm

```diff
@@ -77,18 +77,14 @@
 	siemens_coefficient = pick(0,0.5,0.5,0.5,0.5,0.75,1.5)
 
 /obj/item/clothing/gloves/black
-	desc = "These gloves are fire-resistant."
+	desc = "These gloves are quite insulative!"

```

Comment:
You may want to reword this so people don't confuse it with electrical insulation.